soc_limits.py

```diff
@@ -0,0 +1,8 @@
+SOC_MIN = 20
+SOC_MAX = 80
+
+def soc_is_ok(soc):
+    if soc < SOC_MIN or soc > SOC_MAX:
```

Comment:
Similar code is used for temperature check too. There is scope for reducing code duplication here.